### ISSUE TITLE
No RGB Matrix library found...

### ISSUE THREAD

**Issue by blanddnalb** [partofthething/infopanel#3]
@partofthething 

Forgive me but I have the RPI-RGB-LED-MATRIX api working with demos and such.  However, I saw your code and am trying to get it to work.  I am getting: No RGB Matrix library found. Cannot use that display

Am I missing something?  It looks like it calls for rgbmatrix but can't find it?

**Comment by partofthething** [comment]
For some reason, the dependency isn't getting found in your import path. How did you install the [rpi-rgb-led-matrix dependency](https://github.com/hzeller/rpi-rgb-led-matrix/tree/master/bindings/python)? When I did it, I believe I went into the `bindings/python` folder and ran something like `make install` or `python setup.py install`. As long as this library is either installed this way or pointed to in your `PYTHONPATH` environment variable, the import should succeed. 

If you have it installed normally, it does look like the library has upgraded a few times so it's possible that infopanel has compatibility issues with their latest version.
